CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# dbt_netsuite v0.14.0
+
+For Netsuite2, [PR #138](https://github.com/fivetran/dbt_netsuite/pull/138) and [PR #132](https://github.com/fivetran/dbt_netsuite/pull/132) include the following updates: 
+
+## Breaking Changes (Full refresh required after upgrading)
+- Partitioned models have had the `partition_by` logic adjusted to include a granularity of a month. This change should only impact BigQuery warehouses and was applied to avoid the common `too many partitions` error users have experienced due to over-partitioning by day. Therefore, adjusting the partition to a monthly granularity will increase the partition windows and allow for more performant querying. 
+- This change was applied to the following models:
+  - `int_netsuite2__tran_with_converted_amounts`
+  - `netsuite2__balance_sheet`
+  - `netsuite2__income_statement`
+  - `netsuite2__transaction_details`
+
+## Upstream Netsuite Source Breaking Changes (Full refresh required after upgrading)
+- Casted specific timestamp fields across all staging models as dates where the Netsuite UI does not perform timezone conversion. Keeping these fields as type timestamp causes issues in reporting tools that perform automatic timezone conversion.   
+- Adds additional commonly used fields within the `stg_netsuite2__*` models. **IMPORTANT**: Nearly all of these models where fields are being added have pass-through functionality. So you will need to remove these fields from your passthrough variable setup if they are currently present to avoid errors.
```

Comment:
also which end models had new fields added to them? could be good to highlight here (or are they in the feature updates section?)

Reply:
This will be called out in the `dbt_netsuite_source` release since the changes are being made there. 

I linked to it in line 17, but I updated the wordage to make it clearer where to find these fields.